MIT.md

```diff
@@ -1,6 +1,7 @@
-## Copyright 2021, [YOUR NAME]
+## Copyright 2021, Denis Obote
 
 ###### Please delete this line and the next one
+
 ###### APP TYPE can be a webpage/website, a web app, a software and so on
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
```

Comment:
- [ ] As mentioned previously pelase make sure to replace all the placeholder texts in the MIT file with the project-appropriate content as I noticed the "[APP TYPE]" placeholder text hasn't been replaced, you can for example specify "WEB APP". I'd also suggest updating the year with the current date.
  ![image](https://github.com/Obote/Leaderboard/assets/104968236/b2bb0049-5b6c-40e2-af0b-1c921fc53a3a)